src/rpc/methods/eth.rs

```diff
@@ -3169,6 +3174,99 @@ impl RpcMethod<2> for EthTraceReplayBlockTransactions {
     }
 }
 
+fn get_eth_block_number_from_string<DB: Blockstore>(
+    chain_store: &ChainStore<DB>,
+    block: Option<&str>,
+) -> Result<EthUint64> {
+    let block_param = match block {
+        Some(block_str) => ExtBlockNumberOrHash::from_str(block_str)?,
+        None => bail!("cannot parse fromBlock"),
+    };
+    Ok(EthUint64(
+        tipset_by_ext_block_number_or_hash(chain_store, block_param)?.epoch() as u64,
+    ))
+}
+
+pub enum EthTraceFilter {}
+impl RpcMethod<1> for EthTraceFilter {
+    const N_REQUIRED_PARAMS: usize = 1;
+    const NAME: &'static str = "Filecoin.EthTraceFilter";
+    const NAME_ALIAS: Option<&'static str> = Some("trace_filter");
+    const PARAM_NAMES: [&'static str; 1] = ["filter"];
+    const API_PATHS: ApiPaths = ApiPaths::V1;
+    const PERMISSION: Permission = Permission::Read;
```

Comment:
Let's add the description for all new methods.

Reply:
noted done